### PR TITLE
Removed sqlite3 dependency

### PR DESCRIPTION
**Pull request by dpgraham**
-Uses teen_process exec to run sqlite3 from command line (sqlite3 comes packaged with OSX)
-Inserts sha1 and data in the hex format because they're both BLOBs
-Added a 'hasRecords' and 'getRecordCount' function that checks how many records are in tsettings and then parses the result
-Checks that 'sqlite3' is available on path before installing an SSL cert (just in case it was uninstalled or unavailable in a future release)
-Updated tests to use new code

Reviewers @jlipps @imurchie

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -144,8 +144,16 @@ async function installSSLCert (pemText, udid) {
     log.errorAndThrow(`Command 'openssl' not found`);
   }
 
-  let tempFileName = path.resolve(__dirname, 'temp-ssl-cert.pem');
-  let pathToKeychain = new Simulator(udid).getDir();
+  // Check that sqlite3 is installed on the path
+  try {
+    await exec('which', ['sqlite3']);
```
Comment (imurchie) [inline lib/utils.js]: Does `await fs.which('sqlite3');` not work?

Comment (dpgraham) [inline lib/utils.js]: It does work. I'll update that for consistency.

**Comment by imurchie** [comment]
I think something went wrong with rebasing, etc. Merge commits in there will make merging and squashing weird.

**Comment by jlipps** [comment]
I squashed and merged using github's api! muhaha

**Comment by jlipps** [comment]
published in `appium-simulator@1.11.1`

**Pull request merged**